django_mongodb_backend/gis/adapter.py

```diff
@@ -0,0 +1,27 @@
+import collections
+
+
+class Adapter(collections.UserDict):
+    srid = 4326
+
+    def __init__(self, obj, geography=False):
+        """
+        Initialize on the spatial object.
+        """
```

Comment:
Can you add a link to the `MongoDB GeoJSON Object types` in this docstring?
https://www.mongodb.com/docs/manual/reference/geojson/